dev_guide/builds.adoc

```diff
@@ -84,6 +84,33 @@ The `*ImageChange*` trigger creates a new build each time the Docker image speci
 <4> The `*strategy*` section describes the build strategy used to execute the build. You can specify `*Source*`, `*Docker*` and `*Custom*` strategies here. This specification uses the `*ruby-20-centos7*` Docker image that Source-To-Image will use for the application build.
 <5> After the Docker image is successfully built, it will be pushed into the repository described in the `*output*` section.
 
+== Defining an incremental buildConfig
+Source-To-Image (S2I) builds provide an option to perform incremental build, which means they will
+reuse artifacts from previously built image. To create an incremental build one
+needs to create a BuildConfig as previously explained, but with a slight modification
+in strategy definition as presented below:
+
+[incremental.json]
+----
+{
+  "strategy": {
+    "type": "Source",
+    "sourceStrategy": {
+      "from": {
+        "kind": "ImageStreamTag",
+        "name": "incremental-image <1>"
+      },
+      "incremental": true <2>
+    }
+  }
+}
+----
+
+<1> You need to specify here an image that supports incremental builds. Note: the S2I images provided by OpenShift do not implement artifact reuse, so setting incremental to true will have no effect on BuildConfigs using those builder images.
+<2> This flag controls whether an incremental build is attempted. If the builder image does not support incremental builds, the build will still succeed but you will get log message that the incremental build wasn't successful because of missing `save-artifacts` script.
```

Comment:
s/get log message/get a log message/

s/wasn't/was not/

s/because of missing/because of a missing/